seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-common/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/AbstractJdbcIT.java

```diff
@@ -496,6 +497,16 @@ public void testCatalogWithCatalogUtils() throws SQLException, ClassNotFoundExce
                                     .getFullName()
                                     .equalsIgnoreCase(jdbcCase.getTablePathFullName()));
                 });
+        Collection<JdbcSourceTable> sourceTables = tables.values();
+        sourceTables.forEach(
+                sourceTable -> {
+                    final String catalogName =
+                            sourceTable.getCatalogTable().getTableId().getCatalogName();
+                    if (this.catalog != null) {
+                        Assertions.assertTrue(
+                                StringUtils.equalsIgnoreCase(catalogName, this.catalog.name()));
+                    }
+                });
```

Comment:
See you made additions in this test, but this test filters the operation of some types of databases through `jdbcCase.getTablePathFullName()`,I think you could supply `tablePathFullName` in other type, or write another test

Reply:
cc @Hisoka-X   Since the default is `default.default.default`, see if the test here needs to be supplemented

Reply:
>  write another test

+1 @chl-wxp